src/main/resources/io/jenkins/plugins/pipelinegraphview/PipelineGraphViewAction/index.jelly

```diff
@@ -13,6 +13,10 @@
           </h1>
         </div>
         <div class="jenkins-app-bar__controls">
+          <a href="../pipeline-console" class="jenkins-button">
+            <l:icon src="symbol-terminal-outline plugin-ionicons-api"/>
+            ${%Console View}
+          </a>
```

Comment:
This is a little redundant since you can click on a node in the graph to navigate to the console view. I thought it was good to add for consistency.